### PR TITLE
Add post: Celebrating Fedora 25 with freenode

### PR DESCRIPTION
**Pull request by jflory7**
This is the blog post that I have discussed with @svbeon via email for the past few months. It took me longer to get this together than I would have liked, but now that my exams are finished, I'm hoping that guest articles can be something we continue to work together on over time!

I'm going to ping a few other Fedora folks to check this out, but please let me know if you have any feedback or if any other changes are needed. I figure the date / time stamp of the post may need to be changed, depending on when we are ready to merge, but that should be an easyfix.

### CODE REVIEW

File: content/news/2016-12-17-celebrating-fedora-freenode.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+Title: Celebrating Fedora 25 with freenode
+Author: Justin W. Flory (jflory7)
+Date: 2016-12-17T14:49-05:00
+Slug: celebrating-fedora-freenode
+---
+
+![Celebrating Fedora 25 with freenode](https://i.imgur.com/yMLgxZp.png)
```
Comment (jflory7) [inline content/news/2016-12-17-celebrating-fedora-freenode.md]: @svbeon If preferred, I can change the URL to the eventual location of the image on the freenode site.

Comment (svbeon) [inline content/news/2016-12-17-celebrating-fedora-freenode.md]: @jflory7 change it to Celebrating Fedora 25 with freenode

**Comment by christeld** [comment]
I'm happy for you guys to merge whenever you're ready! It is perfect! Thank you :)

**Comment by svbeon** [comment]
Lovely! Whenever you're ready then :)

File: content/news/2016-12-17-celebrating-fedora-freenode.md (hunk 1 above)
Comment (jflory7) [inline content/news/2016-12-17-celebrating-fedora-freenode.md]: @svbeon Thanks, this is done now!

**Comment by jflory7** [comment]
@svbeon Should have one or two more people giving a quick look today, so should know real soon!

**Comment by svbeon** [comment]
@jflory7 we might have to delay publishing a few days, we want to make sure as many as possible see <http://freenode.net/news/PSA-brief-update-infra>, and we also want to make sure as many as possible see this post, thus it might be a good idea to spread them out in time

**Comment by jflory7** [comment]
@svbeon Yeah, that makes sense. The content should all be set to publish. Maybe do we want to aim for next Wednesday, Dec. 28th, 2016?

**Comment by svbeon** [comment]
Oh actually I was more thinking tomorrow, if that's fine?

**Comment by jflory7** [comment]
@svbeon Yup, that will work fine too. I've modified the timestamp in the file and adjusted the file name accordingly for tomorrow. I also made a quick edit from some other feedback I received. On my end, should be all ready to go! Thanks for your time and patience. 😄

**Pull request merged**